src/utils/utils.js

```diff
@@ -76,3 +78,13 @@ function detach(elem) {
 
   return ans;
 }
+
+/**
+ * Verifies if a given number is power of two or not.
+ * @param {object} n - The number to check.
+ * @return {Array} Returns true if the given number is power of 2, false otherwise.
+ **/
+export function isPowerOfTwo(n) {
```

Comment:
Thought - if we don't need this in multiple places, then maybe put inside texture.js?